docs/driver-overview.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Hardware Drivers
+subtitle: Controlling ev3dev devices through the low-level driver APIs
```

Comment:
I believe most of our pages use a sentence-case subtitle currently (e.g. <http://wasabifan.github.io/ev3dev.github.io/news/>). I personally like this style, for reasons similar to why many news organizations are now sentence-casing their titles: it seems more conversational.